azurerm/data_source_app_service_plan.go

```diff
@@ -73,6 +73,11 @@ func dataSourceAppServicePlan() *schema.Resource {
 				Computed: true,
 			},
 
+			"maximum_elastic_worker_count": {
```

Comment:
Could we add this to the `testAccDataSourceAppServicePlan_complete` test?

Reply:
I added a new test here because it's a new SKU and Kind